test/functional/cmdLineTests/pltest/playlist.xml

```diff
@@ -367,4 +367,34 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<subset>11+</subset>
 		</subsets>
 	</test>
+	<test>
+		<testCaseName>cmdLineTester_pltest_j9sig_ext</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) \
+	$(JAVA_COMMAND) $(JVM_OPTIONS) -DRESOURCES_DIR=$(Q)$(RESOURCES_DIR)$(Q) -DJVM_TEST_ROOT=$(Q)$(JVM_TEST_ROOT)$(Q) \
+	-DEXE=$(SQ)$(JAVA_SHARED_LIBRARIES_DIR)$(D)pltest -inlcude:j9sig_extended $(JVM_OPTIONS)$(SQ) -jar $(CMDLINETESTER_JAR) \
+	-config $(Q)$(TEST_RESROOT)$(D)pltest.xml$(Q) -verbose -explainExcludes \
+	-xids all,$(PLATFORM),$(VARIATION) -plats all,$(PLATFORM),$(VARIATION) -nonZeroExitWhenError; \
+	$(TEST_STATUS)</command>
+		<!-- temporarily disable this test on windows; https://github.com/eclipse/openj9/issues/3212 -->
```

Comment:
This comment is unrelated to `cmdLineTester_pltest_j9sig_ext `. Please remove it.